src/components/Divider.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const StyledDiv = styled.div`
+  position: absolute;
+  top: ${props => (props.top ? props.top : "auto")};
+  bottom: ${props => (props.bottom ? props.bottom : "auto")};
+  left: ${props => (props.left ? props.left : "auto")};
+  right: ${props => (props.right ? props.right : "auto")};
+
+  &::before {
+    position: absolute;
+    content: "";
+    background-image: linear-gradient(
+      to right,
+      rgba(237, 211, 151, 0),
```

Comment:
Isn't this just `rgba(255, 255, 255, 1)`?

Reply:
Yeah, that's just white.